lib/index.js

```diff
@@ -37,6 +37,7 @@ var types = exports.types = {
   , router: zmq.ZMQ_ROUTER
   , pair: zmq.ZMQ_PAIR
   , stream: zmq.ZMQ_STREAM
+  , immediate: zmq.ZMQ_IMMEDIATE
```

Comment:
You've put this in `types` (these are socket types), and should sit in `opts` (these are the options).